src/style/styles.css

```diff
@@ -0,0 +1,144 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;1,100;1,200;1,300;1,400&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Poppins', sans-serif;
+  list-style: none;
+  text-decoration: none;
+  scroll-behavior: smooth;
+}
+
+header {
+  width: 100%;
+  position: fixed;
+  z-index: 100;
+  background-color: #cfc5c5;
+  display: flex;
+  justify-content: space-between;
+  padding: 0 10% 10px;
+}
+
+header h1 {
+  font-size: 1.5rem;
+  font-weight: 500;
+  color: #000;
+  margin: 10px 0;
+  align-items: center;
+  text-align: center;
+}
+
+nav ul {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  list-style: none;
+  gap: 20px;
+  margin-top: 10px;
+}
+
+.navLinks {
+  color: rgb(12, 11, 11);
+  transition: all 0.3s ease-in;
+}
+
+span {
+  color: rgb(146, 76, 76);
+}
+
+.navLinks:hover {
+  color: rgb(146, 76, 76);
+  cursor: pointer;
+}
+
+section {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  padding: 100px 10%;
+}
```

Comment:
Great design team🤝

- [ ] I would suggest you remove the empty space below the footer so that your design can match the design guidelines.
![below-footer](https://github.com/tchasinga/API-capstone-project/assets/63917725/8c459f27-9620-4a46-9fa8-20c5a4dbcca1)